### PR TITLE
Adjust folder to match quickstart (Quickstart will also be updated to match)

### PR DESCRIPTION
**Pull request by sfc-gh-melnacouzi**
Adjust folder structure so it's easier for people following the quickstart to manually upload files:
https://quickstarts.snowflake.com/guide/native-app-chairlift/index.html?index=..%2F..index#1

I will also update the quickstart in a separate PR to match the new folder structure.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1 @@
-output/**
+**output**
```
Comment (sfc-gh-bdufour) [inline .gitignore]: Why is this required? It seems dangerously generic.

File: app/setup_script.sql [hunk 2]
```diff
@@ -13,36 +13,36 @@ grant application role app_viewer to application role app_admin;
 -- so that we do not clobber state whenever we upgrade the app
 create schema if not exists config_data;
     grant usage on schema config_data to application role app_admin;
-    execute immediate from './config_data/configuration.sql';
+    execute immediate from './sql_lib/config_data-configuration.sql';
```
Comment (sfc-gh-bdufour) [inline app/setup_script.sql:16]: FWIW I did prefer the previous organization in subdirectories. Why is this change needed?

Comment (sfc-gh-melnacouzi) [reply app/setup_script.sql:16]: The manual upload (used in the quickstart) does not support recursive uploads, so it's just a torture to do that manually. That was a compromise until we change the tutorial to use snow cli (instead of just support it)

File: prepare/consumer-data.sql [hunk 3]
```diff
@@ -173,11 +173,11 @@ create or replace task populate_reading_every_minute
 as
     call populate_reading();
 
--- Get some initial data in the readings table
-call populate_reading();
+-- If you would like the data to be populated on a schedule, you can run:
```
Comment (sfc-gh-bgoel) [inline prepare/consumer-data.sql]: Going to get pedantic with my documentation experience thus far:
MIke H advises that we replace all "would like"s with "want".

**Review by sfc-gh-bgoel — APPROVED** [review]
lgtm, conditional to Bruno's comments being resolved.

File: .gitignore (hunk 1 above)
Comment (sfc-gh-melnacouzi) [reply .gitignore]: My bad, I was trying to fix `tree --gitignore`, which kept showing output directory.
I fixed it now.

File: app/setup_script.sql (hunk 2 above)
Comment (sfc-gh-bdufour) [reply app/setup_script.sql:16]: ACK

File: prepare/consumer-data.sql (hunk 3 above)
Comment (sfc-gh-melnacouzi) [reply prepare/consumer-data.sql]: updated.

**Review by sfc-gh-bdufour — APPROVED** [review]
